data/translations/meson.build

```diff
@@ -1,6 +1,7 @@
 # Translations
 ts_files = [
     'de.ts'
+    'nl.ts'
```

Comment:
You need to add a comma after the 'de.ts', otherwise it fails:
```
data/translations/meson.build:4:4: ERROR: Expecting rbracket got string.
    'nl.ts'
    ^
For a block that started at 2,11
ts_files = [
           ^

```

Reply:
Did not know that. Fixed it. Thanks! 🙂